bot.go

```diff
@@ -61,70 +72,29 @@ func main() { /*
 	discord.AddHandler(serverJoin)
 	discord.AddHandler(serverLeave)
 	discord.AddHandler(onReady)
-	err = botInit()
-	if err != nil {
-		fmt.Println("Error in botinit:", err)
-		return
-	}
-	if err = discord.Open(); err != nil {
+	if err := discord.Open(); err != nil {
 		fmt.Println("Error opening discord", err)
 		return
 	}
+	defer discord.Close()
 	defer db.Close()
 	fmt.Println("Bot is now running.  Press CTRL-C to exit.")
 	sc := make(chan os.Signal, 1)
-	signal.Notify(sc, syscall.SIGINT, syscall.SIGTERM, os.Interrupt, os.Kill)
-	select {
-	case sig := <-sc:
-		fmt.Println("Quiting due to signal", sig)
-	}
-	discord.Close()
+	signal.Notify(sc, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
I kept os.Interrupt, but I trust you took it out for a reason.

Reply:
I should have actually used os.Interrupt and os.Kill here. It's sematically the same: [the os versions are equal to these two](https://pkg.go.dev/os#Signal)